src/library.js

```diff
@@ -3495,6 +3495,8 @@ LibraryManager.library = {
   },
 
   emscripten_console_log__sig: 'vi',
+  // TODO: Do not merge.
+  emscripten_console_log__proxy: 'sync',
```

Comment:
Talked about this in our meeting: https://github.com/emscripten-core/emscripten/pull/14805

Looks like this is an issue with node and not being able to use `console.log` on worker threads. Should we go ahead with using `out()` and `err()` instead of `emscripten_console_log` or `console_console_warn`.

We could also just introduce a `emscripten_console_out` that Sam suggested. @kripken @tlively

Reply:
How does the issue with `console.log` relate to the rest of this PR? Can it be handled in a separate PR?

Reply:
Yep, this can be split out to a separate PR. 👍